zh-CN/mapping_dev.zh_CN

```diff
@@ -1342,392 +1342,392 @@ dlgSgRenameFile.inf=输入文件的新路径和名称。
+wnd(Compare|ConflictSolver|Git.indexEditor).mniFile-export-html=导出为 HTML 文件
 wnd(Compare|ConflictSolver|Git.indexEditor).mniFile-open-base=打开基本文件更改
 wnd(Compare|ConflictSolver|Git.indexEditor).mniGoto-next-conflict=下一处冲突
 wnd(Compare|ConflictSolver|Git.indexEditor).mniGoto-next-diff=下一处差异
 wnd(Compare|ConflictSolver|Git.indexEditor).mniGoto-previous-conflict=上一处冲突
 wnd(Compare|ConflictSolver|Git.indexEditor).mniGoto-previous-diff=上一处差异
 wnd(Compare|ConflictSolver|Git.indexEditor).mniShow-line-numbers=显示行号
 wnd(Compare|ConflictSolver|Git.indexEditor).mniView-ignore-whitespaces-all=忽略所有空白字符变化
-wnd(Compare|ConflictSolver|Git.indexEditor).mniView-ignore-whitespaces-leading-trailing=Ignore Leading/Trailing Whitespaces for Line Comparison
-wnd(Compare|ConflictSolver|Git.indexEditor).mniView-ignore-whitespaces-none=Ignore No Whitespace for Line Comparison
+wnd(Compare|ConflictSolver|Git.indexEditor).mniView-ignore-whitespaces-leading-trailing=忽略前导/尾随空格以进行行比较
+wnd(Compare|ConflictSolver|Git.indexEditor).mniView-ignore-whitespaces-none=忽略没有用于行比较的空格
 wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-all=所有
-wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-beside-right=Left Beside Right
-wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-merge=Left and Merge
-wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-over-right=Left Above Right
-wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-right-above-merge=Left and Right Above Merge
-wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-right-merge=Merge and Right
+wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-beside-right=左边右边
+wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-merge=左和合并
+wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-over-right=左上方
+wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-right-above-merge=左右合并
+wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-right-merge=合并和右
 wnd(Compare|ConflictSolver|Git.indexEditor).mniView-refresh=刷新
 wnd(Compare|ConflictSolver|Git.indexEditor).mniView-remember-as-default=记住默认
 wnd(Compare|ConflictSolver|Git.indexEditor).mniView-settings=设置
-wnd(Compare|ConflictSolver|Git.indexEditor).mniView-show-current-line-control=Show Long Current Lines
+wnd(Compare|ConflictSolver|Git.indexEditor).mniView-show-current-line-control=显示长的当前行
 wnd(Compare|ConflictSolver|Git.indexEditor).mniView-synchronize-scrolling=同步滚动
 wnd(Compare|ConflictSolver|Git.indexEditor).tbr"Next Change"=下一处差异
 wnd(Compare|ConflictSolver|Git.indexEditor).tbr"Prev. Change"=上一处差异
 wnd(Compare|ConflictSolver|Git.indexEditor).tbr"Reload"=重新加载
-wnd(Compare|ConflictSolver|Git.indexEditor).tbr"Take Left"=Take Left
-wnd(Compare|ConflictSolver|Git.indexEditor).tbr"Take Right"=Take Right
+wnd(Compare|ConflictSolver|Git.indexEditor).tbr"Take Left"=向左
+wnd(Compare|ConflictSolver|Git.indexEditor).tbr"Take Right"=向右
```

Comment:
"使用左边文件块" more suitable? There are other similar take.

Reply:
I think "左边" would be more appropriate.

Reply:
Translation that is consistent with the tools of difference comparison

Reply:
> Translation that is consistent with the tools of difference comparison

The "左边", I think the 'take' is not reflected, because "Take Left" is an action, "take" is more important.

And "使用左边文件块" is using another git GUI tool, TortoiseGit. So i think "使用左边文件块" is more suitable. 

Do you think?

Reply:
```
wnd(Compare|ConflictSolver|Git.indexEditor).mniEdit-take-left=Take Left Block
wnd(Compare|ConflictSolver|Git.indexEditor).mniEdit-take-left-right=Take Left, then Right Block
wnd(Compare|ConflictSolver|Git.indexEditor).mniEdit-take-right=Take Right Block
wnd(Compare|ConflictSolver|Git.indexEditor).mniEdit-take-right-left=Take Right, then Left Block

wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-merge=HEAD and Index
wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-over-right=Left Above Right
wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-right-above-merge=HEAD and Working Tree Above Index
wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-right-merge=Index and Working Tree
```
When comparing with the original text, I find that there are some differences between the two versions.

Reply:
I don't understand. What do you mean? 

The upper half is action, the second half just layout.

Reply:
Here I translate with reference to the latest original text.

Reply:
OK, Maybe @mstrap was changed.

Reply:
For

``
wnd(Compare|ConflictSolver|Git.indexEditor).mniEdit-take-left=Take Left Block
wnd(Compare|ConflictSolver|Git.indexEditor).mniEdit-take-left-right=Take Left, then Right Block
wnd(Compare|ConflictSolver|Git.indexEditor).mniEdit-take-right=Take Right Block
wnd(Compare|ConflictSolver|Git.indexEditor).mniEdit-take-right-left=Take Right, then Left Block
``

I can't see any differences between latest `mapping` and `mapping_dev.zh_CN`. @taotieren, do I miss something? If so, can you please point out more detailed?

For

``
wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-beside-right=Left Beside Right
wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-merge=HEAD and Index
wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-over-right=Left Above Right
wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-left-right-above-merge=HEAD and Working Tree Above Index
wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout-right-merge=Index and Working Tree
``

there is actually a problem. I will fix this soon. Please wait with translation.

Reply:
With respect to `wnd(Compare|ConflictSolver|Git.indexEditor).mniEdit-take-left`, please note that this is different from `wnd(Compare|ConflictSolver|Git.indexEditor).tbr"Take Left"` (which the diff refers to).

Reply:
Because the Chinese translation before this place is different from the latest original text, I found something different when comparing the original text. I re-translated this part.

Reply:
> With respect to `wnd(Compare|ConflictSolver|Git.indexEditor).mniEdit-take-left`, please note that this is different from `wnd(Compare|ConflictSolver|Git.indexEditor).tbr"Take Left"` (which the diff refers to).

Thanks, I've fixed it.

Reply:
I have now fixed `wnd(Compare|ConflictSolver|Git.indexEditor).mniView-layout*`-keys: it was wrong to have a single entry for all windows; actually these are different "actions" and thus must be separated.